charts/dashboard/templates/deployment.yaml

```diff
@@ -31,7 +31,7 @@ spec:
       terminationGracePeriodSeconds: 60
       containers:
         - name: dashboard
-          image: "{{ required ".Values.image.repository missing" .Values.image.repository }}:{{ required ".Values.image.tag missing" .Values.image.tag }}"
+          image: "{{ required ".Values.image.repository missing" .Values.image.repository }}@{{ required ".Values.image.digest missing" .Values.image.digest }}"
```

Comment:
Let's keep it as `tag`. This is  The digest used is technically also a tag on the container image.

Reply:
We need the image digest to enable redeployment because using the latest tag does not trigger any changes in the deployment, even if a new image is tagged as latest. Additionally, using the digest provides a clearer mapping to the specific version being deployed.